bin/console

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require "bundler/setup"
+require "mais_orcid/client"
```

Comment:
Is this right? Should it be `require "mais_orcid_client"`

Reply:
Not right. Fixed now.